2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,31 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    return (this.volume = 100);
   },
-  pour: function () {
+  pour: function (units) {
```

Comment:
There should be no need to declare a parameter here, when the function `pour()` is called, it will increase by 10 units each time if the defined condition is met